code/game/alternate_appearance.dm

```diff
@@ -13,6 +13,9 @@ GLOBAL_LIST_EMPTY(active_alternate_appearances)
 		return
 	if(alternate_appearances && alternate_appearances[key])
 		return
+	if(!(type in subtypesof(/datum/atom_hud/alternate_appearance)))
```

Comment:
might be worth static caching the subtype list

Reply:
I mean 
![image](https://user-images.githubusercontent.com/58055496/135985860-7c5553c2-d1a6-4910-810f-9b648daad532.png)